### PR TITLE
Added database migrations to the toolchain

### PR DESCRIPTION
**Pull request by skeswa**
This is one step toward getting everything working in staging.

### CODE REVIEW

File: common/config/cli.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	"bytes"
+	"os"
+	"strconv"
+
+	"gopkg.in/urfave/cli.v1"
+)
+
+const (
+	environmentDev  = "dev"
+	environmentProd = "prod"
```
Comment (Shikkic) [inline common/config/cli.go]: Nice!

Comment (Shikkic) [inline common/config/cli.go]: Oh snap I did not realize you refactored the gophrcli tooling to be a go cli program, vry nice.

File: common/errors/no_such_package.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package errors
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// NoSuchPackageError is an error that occurs when a requested package doesn't
+// exist.
+type NoSuchPackageError struct {
```
Comment (Shikkic) [inline common/errors/no_such_package.go]: Nice! I needed this for the indexer!

**Comment by Shikkic** [comment]
Looks great! This is definitely going to break my PR. Let's merge this as soon as possible so I can refactor mine.

File: common/config/cli.go (hunk 1 above)
Comment (skeswa) [inline common/config/cli.go]: I didn't. That's for loading configuration from the environment in `api`, `router`, `indexer`, etc.

**Comment by Shikkic** [comment]
Ayy lmao, nice work man! I will re-submit the PR for new indexer tonight/tomorrow.

**Pull request merged**